src/boolean/randomBool.ts

```diff
@@ -0,0 +1,14 @@
+import { randomInt } from '../number';
```

Comment:
The file extension is required for the ESM support.

```suggestion
import { randomInt } from '../number.ts';
```